### ISSUE TITLE
onChangeComplete Doesn't really work when color drag is completed

### ISSUE THREAD

**Issue by njofce** [scttcper/ngx-color#288]
There is an issue with onChangeComplete. Try dragging the color picker around, or just the opacity indicator, and it gets triggered multiple times. It should only be triggered onDragEnd

**Comment by scttcper** [comment]
its a debounced event as opposed to a rapid fire, this is how both react-color and this library implement the onChangeComplete

**Comment by njofce** [comment]
@scttcper 
Yes, but I think it's not the desired behavior. I checked the entire library, and I guess some major changes are required to support this. I would like to try a PR, but please let me know if you have a better/faster solution for this.

**Comment by KlausHans** [comment]
I agree. I would expect, that onChangeComplete only triggers if the user releases the mouse button (or raises his finger).

**Comment by scttcper** [comment]
There's probably room for an onMouseUp event
